### DISCUSSION TITLE
MonitorControl crashes with a trace trap message when mac comes out of monitor sleep

### DISCUSSION THREAD

**Discussion by ivahos** [MonitorControl/MonitorControl#1063] [Q&A]
Hi
I have discovered a bug when running monitorcontrol on my mac studio. Sometimes the mac uses a longer time than normal to detect the monitor after a monitor sleep, when this happens MonitorControl often crashes. 
ivar@Jumbo MacOS % ./MonitorControl
zsh: trace trap  ./MonitorControl

I have created a workaround by starting the app from a shell script that keeps relaunching it whenever it quits.
[MonitorControl-2022-06-03-111213.ips.zip](https://github.com/MonitorControl/MonitorControl/files/8828935/MonitorControl-2022-06-03-111213.ips.zip)
I am attaching the crash log from the console app.

**Comment by waydabber** [comment]
There are a number of these crash reports. We should devote some time to fix this issue indeed.

**Reply by waydabber** [reply]
(created an issue - #1065)
